### PR TITLE
DSS-40: Hero

### PR DESCRIPTION
**Pull request by zastrow**

### CODE REVIEW

File: src/components/hero.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div className="cmp-hero">
+  <h1 className="cmp-hero__heading">
+    <a className="cmp-hero__logo" href="https://seesparkbox.com">
+      <img src="/images/logo.svg" alt="Sparkbox" />
+    </a>
+    <span className="cmp-hero__title">Design Systems</span>
+    <span className="cmp-hero__survey">Survey</span>
+    <span className="cmp-hero__year">2019 Edition</span>
+  </h1>
+  <div className="cmp-hero__flood" role="image"></div>
```
Comment (mdt2) [inline src/components/hero.mdx]: The validator is not liking these `role="image"`: 
Screen Shot 2019-07-17 at 10 45 05 AM

**Comment by mdt2** [comment]
Looks great in modern browsers! 👏 Could use some work in the older browsers. This is what I'm seeing in IE11. It's similar in Edge, and even more jumbled in IE9. 
Screen Shot 2019-07-17 at 3 54 09 PM

**Comment by zastrow** [comment]
Gonna be honest. I didn’t expect it to be that bad in IE11. On it.

File: src/scss/_components.hero.scss [hunk 2]
```diff
@@ -222,17 +261,19 @@ $z-heading: 600;
       display: block;
       position: absolute;
       @include positions(0);
-      background: url("/images/lightning-dim.png") no-repeat top right scroll / auto 100%;
+      background: url("/images/lightning-dim.png") no-repeat top right;
+      background-size: auto 100%;
     }
 
-    @supports (mix-blend-mode: screen){
+    @supports (mix-blend-mode: overlay){
```
Comment (mdt2) [inline src/scss/_components.hero.scss]: Should this be `screen`? Just wondering since we're not using `overlay` in this code block.

**Pull request merged**